src/Evolution/Systems/GrMhd/ValenciaDivClean/Fluxes.hpp

```diff
@@ -0,0 +1,64 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"  // IWYU pragma: keep
+
+/// \cond
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+
+class DataVector;
+/// \endcond
+
+// IWYU pragma: no_forward_declare Tensor
+
+namespace grmhd {
+namespace ValenciaDivClean {
+
+/*!
+ * \brief The fluxes of the conservative variables
+ *
+ * \f{align*}
+ * F^i({\tilde D}) = &~ {\tilde D} v^i_{tr} \\
+ * F^i({\tilde S}_j) = &~  {\tilde S}_j v^i_{tr} + \sqrt{\gamma} \alpha \left( p
+ * + p_m \right) \delta^i_j - \frac{B_j {\tilde B}^i}{W^2} - v_j {\tilde B}^i
+ * B^m v_m\\
+ * F^i({\tilde \tau}) = &~  {\tilde \tau} v^i_{tr} + \sqrt{\gamma} \alpha \left(
+ * p + p_m \right) v^i - \alpha {\tilde B}^i B^m v_m \\
+ * F^i({\tilde B}^j) = &~  {\tilde B}^j v^i_{tr} - \alpha v^j {\tilde B}^i +
+ * \alpha \gamma^{ij} {\tilde \Phi} \\ F^i({\tilde \Phi}) = &~ \alpha {\tilde
+ * B^i} - \beta^i {\tilde \Phi} \f} where the conserved variables \f${\tilde
+ * D}\f$, \f${\tilde S}_i\f$, \f${\tilde \tau}\f$, \f${\tilde B}^i\f$, and
+ * \f${\tilde \Phi}\f$ are a generalized mass-energy density, momentum density,
+ * specific internal energy density, magnetic field, and divergence cleaning
```

Comment:
I find the two list combined sentence really long. Could you split it after then `divergence cleaning field`?